samples/hello-oboe/src/main/java/com/google/oboe/samples/hellooboe/MainActivity.java

```diff
@@ -106,6 +111,11 @@ protected void onResume() {
         mBufferSizeSpinner.setSelection(SPINNER_DEFAULT_OPTION_INDEX);
         mAudioApiSpinner.setSelection(SPINNER_DEFAULT_OPTION_INDEX);
 
+        if (mScoStarted) {
```

Comment:
Can mScoStarted be true if onPause() works correctly?

Reply:
In the code directly above, onResume sets the options back to the defaults. Thus, we should reset this when this happens

Reply:
In the code directly above, onResume sets the options back to the defaults. Thus, we should reset this when this happens